gum/arch-arm/gumarmbacktracer.c

```diff
@@ -131,7 +131,8 @@ gum_arm_backtracer_generate (GumBacktracer * backtracer,
       {
         if (value % 4 == 0)
         {
-          const guint32 insn = *((guint32 *) GSIZE_TO_POINTER (value - 4));
+          const guint32 insn = GUINT32_FROM_LE(
+            *((guint32 *) GSIZE_TO_POINTER (value - 4)));
```

Comment:
```suggestion
              *((guint32 *) GSIZE_TO_POINTER (value - 4)));
```

Hanging indents should be four spaces, not two.